### ISSUE TITLE
SyntaxError: Unexpected token <

### ISSUE THREAD

**Issue by Locutus73** [ilcato/homebridge-melcloud#3]
Hi,
today, whenever homebridge-melcloud tries to get the status of the units (i.e. opening Home app), homebridge crashes with the following output:

```
undefined:3
<!DOCTYPE html>
^

SyntaxError: Unexpected token <
    at Request._callback (/usr/lib/node_modules/homebridge-melcloud/index.js:174:60)
    at Request.self.callback (/usr/lib/node_modules/homebridge-melcloud/node_modules/request/request.js:186:22)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request.<anonymous> (/usr/lib/node_modules/homebridge-melcloud/node_modules/request/request.js:1081:10)
    at emitOne (events.js:96:13)
    at Request.emit (events.js:188:7)
    at IncomingMessage.<anonymous> (/usr/lib/node_modules/homebridge-melcloud/node_modules/request/request.js:1001:12)
    at IncomingMessage.g (events.js:291:16)
    at emitNone (events.js:91:20)
    at IncomingMessage.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)

```

Many thanks in advance,
best regards.

Locutus73

**Comment by kuroshirokun** [comment]
Same issue here

Best regards

**Comment by ilcato** [comment]
There's an issue on melcloud. Sometimes it doesn't respond correctly to the requests with an internal error. I intercepted this case an keep homebridge running.
Just published in version 0.3.3 on npm.
